tests/Doctrine/Migrations/Tests/Tools/Console/Command/MigrateCommandTest.php

```diff
@@ -388,8 +388,8 @@ public static function allOrNothing(): Generator
         yield [true, ['--all-or-nothing' => 0], false];
         yield [true, ['--all-or-nothing' => '0'], false];
 
-        yield [true, [], true];
-        yield [false, [], false];
+        yield [true, [], true, false];
```

Comment:
I'm confused on why this test is failing. I should look into it later. Any guidance welcomed :smiley:

Reply:
I faintly remember this `notprovided` is here for a reason. Can you look through the history of commits/PRs/issues to find it?

Reply:
@gaetan-petit I made an [alternative PR](https://github.com/doctrine/migrations/pull/1381) with these test changes.

TL;DR as part of the deprecation mechanism, I overlooked checking that the deprecation was not thrown. I adjusted the logic as well to achieve the desired result. Please give a review and point out if there's any flaw in the logic changes.

cc @greg0ire feel free to merge if it's ok for you.

Reply:
@agustingomes indeed your pr is a better solution. I am closing this one.